cloud

```diff
@@ -1764,7 +1769,7 @@ function execute_command() {
     eval "declare -a argv=($argv)"
 
     declare -r load_queue_trigger_topic="${prefix}load_queue_trigger"
-    declare -r storage_bucket="${project}_${prefix}cache"
+    declare -r cache_bucket_name="${project}_${prefix}cache"
```

Comment:
I might've misunderstand what Nik meant by "not via shared namespace", but I guess it includes removing this definition (given that `--prefix="${namespace:-}${namespace:+_}kcidb_" \`)

You can see that `cache_bucket_name` is already defined via getopt_vars a few lines above...
Keeping this line here you should see that the bucket was using the previous naming scheme.

Reply:
Thanks, @tales-aparecida. We were not supposed to have the `cache_bucket_name` as a parameter to this function. It's supposed to be generated right here, as written above.

What I [meant](https://github.com/kernelci/kcidb/pull/439#discussion_r1271876093) by "not via shared namespace" was that @octonawish-akcodes was not passing the `storage_bucket` to the `cloud_functions_env()` function explicitly (it didn't have it as a parameter), and his code only worked because in shell a function sees the variables declared in the calling function :see_no_evil:

Reply:
Yeah it just take it from the environment of that instance after I exported it.